spec/controllers/admin/videos_controller_spec.rb

```diff
@@ -0,0 +1,137 @@
+require 'spec_helper'
+
+describe Admin::VideosController do
+
+  include CarrierWave::Test::Matchers
+
+  describe 'get :new' do
+
+    before(:each) do
+      admin = Fabricate(:admin)
+      login_user admin
+    end
+
+    it_behaves_like 'requires sign in' do
+      let(:action) { get :new }
+    end
+
+    it_behaves_like 'requires admin' do
+      let(:action) { get :new }
+    end
+
+    it 'sets @video variable' do
+      get :new
+      expect(assigns(:video)).to be_instance_of(Video)
+      expect(assigns(:video)).to be_new_record
+    end
+
+  end
+
+  describe 'post :create' do
+
+    it_behaves_like 'requires sign in' do
+      let(:action) { post :create }
+    end
+
+    it_behaves_like 'requires admin' do
+      let(:action) { post :create, video: {} }
+    end
+
+    context 'for valid values' do
+
+      let(:category) { Fabricate(:category) }
+      before(:each) do
+        admin = Fabricate(:admin)
+        login_user admin
+      end
+
+      context 'without images' do
+
+        let(:video_attrs) { Fabricate.attributes_for(:video, category_id: category.id) }
```

Comment:
This doesn't need to be a `let`. Since you only use it in the `before` block, you can move it there.